share/sensitivity-analysis/cyclus_dakota.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import os
+import input as inp
+import dakota.interfacing as di
+import subprocess
+# ----------------------------
+# Parse Dakota parameters file
+# ----------------------------
+
+params, results = di.read_parameters_file()
+
+# -------------------------------
+# Convert and send to Cyclus
+# -------------------------------
+
+# Edit Cyclus input file
+cyclus_template = 'test.xml.in'
+scenario_name = 'PW' + str(round(params['power']))
+variable_dict = {'handle': scenario_name, 'power_cap': params['power']}
+output_xml = 'test.xml'
+inp.render_input(cyclus_template, variable_dict, output_xml)
+
+# Run Cyclus with edited input file
+output_sqlite = 'test'+scenario_name+'.sqlite'
+os.system('cyclus -i ' + output_xml + ' -o ' + output_sqlite)
+
+# ----------------------------
+# Return the results to Dakota
+# ----------------------------
+
+for i, r in enumerate(results.responses()):
```

Comment:
why enumerate? You don't use the enumeration `i`.

Reply:
This structure is in the test examples provided by Dakota. It becomes useful when you have >1 number of output functions.